### PR TITLE
mgr/volumes/fs: add extra blank line

### PR DESCRIPTION
**Pull request by Astroakanksha24**
## Checklist
- [x] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

Fixes: https://tracker.ceph.com/issues/51393
Fixes: https://tracker.ceph.com/issues/51385
Signed-off-by: Akanksha Chaudhari <astroakanksha24@gmail.com>

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Comment by Astroakanksha24** [comment]
I am solving this issue under OSD

**Comment by varshar16** [comment]
Hi Akansha, please see the submitting patches doc and update the patch with `git commit --amend`.

**Comment by varshar16** [comment]
Also add this tracker to the patch: https://tracker.ceph.com/issues/51393

**Review by batrick — CHANGES_REQUESTED** [review]
Please [sign your commits](https://github.com/ceph/ceph/blob/master/SubmittingPatches.rst#sign-your-work). This is required for your PR to be merged.

See [this article](https://help.github.com/en/articles/changing-a-commit-message) on GitHub on how to amend commits and update your pull request.

**Comment by Astroakanksha24** [comment]
Done.

**Comment by Astroakanksha24** [comment]
@varshar16  please check this one!

File: src/pybind/mgr/volumes/fs/operations/group.py [hunk 1]
```diff
@@ -181,10 +185,12 @@ def open_group(fs, vol_spec, groupname):
...
 
+
 @contextmanager
 def open_group_unique(fs, vol_spec, groupname, c_group, c_groupname):
     if groupname == c_groupname:
         yield c_group
     else:
         with open_group(fs, vol_spec, groupname) as group:
             yield group
+
```
Comment (varshar16) [inline src/pybind/mgr/volumes/fs/operations/group.py:196]: Remove this extra line. See [this](https://www.flake8rules.com/rules/E302.html) to understand why the extra blank line is required.

**Review by varshar16 — CHANGES_REQUESTED** [review]
The commits are little messed up. Please rebase the commit by `git rebase origin/master`,  update the patch with `git commit --amend` and push amended changes using `git push -f`. Check `git log` or history of this file to get ideas about commit title and message.

**Comment by varshar16** [comment]
@Astroakanksha24 you need to still rebase your patch but the final commit looks good. The commit message needs to be
``
Fixes: https://tracker.ceph.com/issues/51385
Signed-off-by: Akanksha Chaudhari <astroakanksha24@gmail.com>
``

**Comment by varshar16** [comment]
The commit title is wrong. It needs to be `mgr/volumes/fs/operations/group: add extra blank line`

**Comment by varshar16** [comment]
Please fix up the commit message of this commit ca5a9a7
```
Fixes: https://tracker.ceph.com/issues/51385
Signed-off-by: Akanksha Chaudhari <astroakanksha24@gmail.com>
```
And in other replace the tracker ticket with https://tracker.ceph.com/issues/51393

**Comment by varshar16** [comment]
There are extra commits. Please rebase your PR

**Comment by ideepika** [comment]
@Astroakanksha24 I guess this might be helpful, for correcting the history right now I see unrelated commits in this PR:

https://stackoverflow.com/questions/64288648/git-history-screwed-up
1. Once you have pre rebased state, to do correct rebase you need to pass "a commit prior to the one you want to change things"
2. please squash and add related `Fixes:` line as @varshar16 suggested

HTH

**Pull request closed without merging**